setup.py

```diff
@@ -20,13 +23,13 @@
     long_description_content_type="text/markdown",
     license="MIT",
     url="https://github.com/offish/steam_community_market",
-    download_url="https://github.com/offish/steam_community_market/tarball/v" + version,
+    download_url=f"https://github.com/offish/steam_community_market/tarball/v{version}",
     packages=["steam_community_market"],
     classifiers=[
         "Programming Language :: Python :: 3",
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
     ],
     install_requires=["requests"],
-    python_requires='>=3.6',
+    python_requires=">=3.6",

```

Comment:
This is no longer correct due to the usage of the new union syntax

Reply:
Correct, modified to `>=3.10`.